App/fetch.js

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
This will not work with "Proxies" as `axios` has issues with `proxy servers`. However just calling it out, unless it is significant, you can ignore it.

Reply:
Oh, right.. We could use **https-proxy-agent** here but I'm afraid of adding more code/complexity.